iOSSampleAppWithSDK/SampleAPP/Podfile.lock

```diff
@@ -0,0 +1,104 @@
+PODS:
+  - Alamofire (5.2.2)
+  - AlamofireImage (4.1.0):
+    - Alamofire (~> 5.1)
+  - Cosmos (17.0.0)
+  - CropViewController (2.4.0)
+  - KeyboardAvoidingView (5.0.1):
+    - ViewState (>= 1.0.0)
+  - LocusLotrSdk (1.0.0):
+    - Alamofire
+    - AlamofireImage
+    - Cosmos (~> 17.0.0)
+    - CropViewController (~> 2.4.0)
+    - KeyboardAvoidingView
+    - Moya/RxSwift
+    - RealmSwift
+    - Reusable
+    - RxCocoa
+    - RxCoreLocation
+    - RxRealm
+    - RxSwift
+  - Moya/Core (14.0.0):
+    - Alamofire (~> 5.0)
+  - Moya/RxSwift (14.0.0):
+    - Moya/Core
+    - RxSwift (~> 5.0)
+  - Realm (5.4.3):
+    - Realm/Headers (= 5.4.3)
+  - Realm/Headers (5.4.3)
+  - RealmSwift (5.4.3):
+    - Realm (= 5.4.3)
+  - Reusable (4.1.1):
+    - Reusable/Storyboard (= 4.1.1)
+    - Reusable/View (= 4.1.1)
+  - Reusable/Storyboard (4.1.1)
+  - Reusable/View (4.1.1)
+  - RxCocoa (5.1.1):
+    - RxRelay (~> 5)
+    - RxSwift (~> 5)
+  - RxCoreLocation (1.5.0):
+    - RxCoreLocation/Core (= 1.5.0)
+  - RxCoreLocation/Core (1.5.0):
+    - RxCocoa (~> 5.1)
+    - RxSwift (~> 5.1)
+  - RxRealm (3.1.0):
+    - RealmSwift (~> 5.2)
+    - RxSwift (~> 5.0)
+  - RxRelay (5.1.1):
+    - RxSwift (~> 5)
+  - RxSwift (5.1.1)
+  - ViewState (1.0.0)
+
+DEPENDENCIES:
+  - "LocusLotrSdk (from `git@github.com:locus-taxy/lotr-sdk-ios.git`, branch `frameWorkUpdate`)"
```

Comment:
Cant we build from master branch itself ?

Reply:
i will make this change once the sdk pr is merged